src/libs/actions/Report.ts

```diff
@@ -726,6 +732,54 @@ function clearAvatarErrors(reportID: string) {
     });
 }
 
+// Add a helper function called getInviteOnboardingDetails to Report. That function will:
+// Read the value of nvp_introSelected
+// Check that it is set, if not return early
+// Check that isInviteOnboardingComplete is false, if not return early
+// Check that inviteType is not iou or invoice, if it is, return early
+// Get the first and last names from personal details
+// Get the correct onboarding message from ONBOARDING_MESSAGES (link)
+// When choice is newDotSubmit we want to call ONBOARDING_MESSAGES[newDotEmployer]
+// When choice is newDotAdmin we want to call ONBOARDING_MESSAGES[newDotAdmin] which will be added here.
+// Optimistically set nvp_introSelected.isInviteOnboardingComplete to true
+// Return the firstName, lastName, and onboardingMessage values
```

Comment:
I don't think we want to keep this, correct?

Reply:
We don't, I overlooked removing this after writing up the proper comment of this function somehow.